powerauth-webflow-authentication/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/service/OperationSessionService.java

```diff
@@ -152,8 +159,9 @@ public boolean registerWebSocketSession(String operationHash, String webSocketSe
             return false;
         }
         String existingWebSocketSessionId = operationSessionEntity.getWebSocketSessionId();
-        if (existingWebSocketSessionId != null && !webSocketSessionId.equals(existingWebSocketSessionId)) {
-            // Registration failed because operation is being accessed from another browser tab or window
+        if (existingWebSocketSessionId != null) {
```

Comment:
Just a question - why did we change this condition? Does this not this cause operation failure during step change? (websocket ID exists and is the one it is supposed to be)

Reply:
The change was done to allow Web Socket registration only once in `startHandshake.js`. We do not need to allow it multiple times. When step changes the Web Socket session ID does not change, the Web Socket is opened only once during initial handshake. The only case when Web Socket would need to re-register would be page refresh which is no longer supported:
  - in case operation ID is known, the operation fails after refresh when user overrides the refresh confirmation dialog
  - in case of implicit operation a new operation is started which can register new Web Socket session because the operation ID is different